dockerImages/runharness/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # Copyright 2017-2019 VMware, Inc.
 # SPDX-License-Identifier: BSD-2-Clause
-FROM namespace/weathervane-centos7:version
+FROM hrosenbe/weathervane-centos7:intraCluster
```

Comment:
Use namespace instead of hrosenbe?
What does intraCluster do?

Reply:
Good catch.  This is something that must have crept in because I aborted a build of the runharness container. intraCluster was what I was using for the version number (in version.txt).